.github/workflows/io-deploy.yaml

```diff
@@ -0,0 +1,86 @@
+name: IO WWW Deploy
```

Comment:
IO app deploy always deploys to prod, but who/when deploys to Stage?
Developer manually?